src/main/java/org/plumelib/util/RandomSelector.java

```diff
@@ -146,7 +147,9 @@ public void accept(T next) {
         @SuppressWarnings("lowerbound:argument") // no list support
         int rem = generator.nextInt(values.size());
         // values should be MinLen(1), meaning that values.size() is positive.
-        values.set(rem, next);
+        if (rem < values.size()) {
```

Comment:
Is it possible for this test to evaluate to `false`?  If so, please add a test where that occurs.